.ci/test_windows.ps1

```diff
@@ -50,3 +74,78 @@ if ($env:TASK -eq "regular") {
   conda install -q -y -n $env:CONDA_ENV ipywidgets notebook
   jupyter nbconvert --ExecutePreprocessor.timeout=180 --to notebook --execute --inplace *.ipynb ; Check-Output $?  # run all notebooks
 }
+
+# test R package
+if ($env:TASK -eq "r-package"){
```

Comment:
I think we should have a separate script file as we do for non-Windows:
https://github.com/microsoft/LightGBM/blob/3a335e9d4f63da10094d690a92cbefd0f4e6faf6/.ci/test.sh#L71-L74

Reply:
oh ok sure! I can do that